lib/housing-wanted/housing-wanted.models.js

```diff
@@ -0,0 +1,45 @@
+const Joi = require('joi');
+const propertyModels = require('../property/property.models.js');
+
+module.exports = {
+  db: (sequelize, Sequelize) => {
+    const HousingWanted = sequelize.define('HousingWanted', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true
+      },
+      author: Sequelize.STRING,
+      title: Sequelize.STRING,
+      location: Sequelize.STRING,
+      beds: Sequelize.INTEGER,
+      baths: Sequelize.INTEGER,
+      contact: Sequelize.STRING,
+      body: Sequelize.TEXT,
+      status: Sequelize.STRING,
+      details: {
+        type: Sequelize.JSONB,
+        defaultValue: {},
+      },
+    });
+
+    // HousingWanted.associate = function (models) {
```

Comment:
we'll need to add author and location as associations here. I can't remember if I told you all to hold off on that, or to go for it. my apologies. We'll go over it in mtg today, but if you want to start on your own, property content type should have an example of author that you can pull from. locations will be slightly more complicated for you and we'll need to create a join/through table with a hasMany association. I think I mentioned this in my other previous comment and linked to the docs section if you want to read up.